polkadot/node/core/approval-voting/Cargo.toml

```diff
@@ -30,6 +31,9 @@ sp-consensus = { path = "../../../../substrate/primitives/consensus/common", def
 sp-consensus-slots = { path = "../../../../substrate/primitives/consensus/slots", default-features = false }
 sp-application-crypto = { path = "../../../../substrate/primitives/application-crypto", default-features = false, features = ["full_crypto"] }
 sp-runtime = { path = "../../../../substrate/primitives/runtime", default-features = false }
+# Needed for migration test helpers
+test-helpers = { package = "polkadot-primitives-test-helpers", path = "../../../primitives/test-helpers" }
```

Comment:
There are some pub functions, like `v1_to_v2_sanity_check`, `v1_to_v2_fill_test_data` which use some functions from `test-helpers`, so it can't be a dev-dependency because there is no way that we put `v1_to_v2_sanity_check` and `v1_to_v2_fill_test_data` under `cfg(test)` because they would have to be used later on in the `parachains_db` tests.

@sandreim, thoughts ?

Reply:
Yeah, this is a miss on my side. After a quick look it one solution could be  duplicating: `dummy_candidate_receipt, dummy_candidate_receipt_bad_sig, dummy_hash` instead of needing test-helpers.

Reply:
Addressed with: https://github.com/paritytech/polkadot-sdk/pull/1178/commits/15eaa0a20f67d3d7569eda39e0ece1bfee8ca6ec